classes.js

```diff
@@ -82,6 +76,15 @@ class Booklet {
   }
 }
 
+// const book = document.querySelector('.book').value;
+// const convertedBookList = JSON.parse(localBookList);
+
+// for (let i = 0; i < convertedBookList.length; i++) {
+//   if (i % 2 === 0) {
+//     book.style.background = "#bebebe";
+//   }
+// }
+
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner